test/cctest/node_test_fixture.h

```diff
@@ -77,7 +82,8 @@ class NodeTestFixture : public ::testing::Test {
   }
 
   virtual void SetUp() {
-    platform_ = v8::platform::CreateDefaultPlatform();
+    uv_loop_init(&current_loop);
```

Comment:
No corresponding `uv_loop_close()` in TearDown()?  As well, you should check the return value.